### PR TITLE
feat: add new multiclass text classification example for tensorflow [DET-5277]

### PR DESCRIPTION
**Pull request by azhou-determined**
adding a new model porting example for tensorflow. 

this is a clean PR from https://github.com/determined-ai/determined/pull/2369 because git amend author on initial commit creates a new commit which still fails CLA bot

### CODE REVIEW

File: examples/nlp/text_classification_tf_keras/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+[word embedding](https://www.tensorflow.org/tutorials/text/word_embeddings) tutorial.
+
+## Files
+* `model_def.py`: The core code for training. This includes building and compiling the model.
+  
+* `data.py`: The data loading and preparation code for the model.
+
+###Configuration Files
+* `const.yaml`: Train the model on a single GPU with constant hyperparameter values.
+* `distributed.yaml`: Same as const.yaml, but trains the model with multiple GPUs (distributed training).
```
Comment (azhou-determined) [inline examples/nlp/text_classification_tf_keras/README.md:16]: > Remove the "distributed training" in parentheses here.
copied comment from @naren-determined

Comment (azhou-determined) [reply examples/nlp/text_classification_tf_keras/README.md]: fixed

File: examples/nlp/text_classification_tf_keras/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+This example demonstrates how to integrate and train a Tensorflow model on Determined's API. 
+We will implement a multiclass classifier that will be trained to perform sentiment 
+analysis on a dataset of Amazon product reviews, classifying reviews into 5 categories (0-5 stars). 
+It is adapted from Tensorflow's
+[word embedding](https://www.tensorflow.org/tutorials/text/word_embeddings) tutorial.
+
+## Files
+* `model_def.py`: The core code for training. This includes building and compiling the model.
+
```
Comment (naren-determined) [inline examples/nlp/text_classification_tf_keras/README.md:11]: Nit: delete line

**Review by naren-determined — APPROVED** [review]

File: examples/nlp/text_classification_tf_keras/README.md (hunk 2 above)
Comment (azhou-determined) [reply examples/nlp/text_classification_tf_keras/README.md]: done

**Pull request merged**